x-pack/plugins/cross_cluster_replication/public/register_ccr_section.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { management } from 'ui/management';
+import { i18n } from '@kbn/i18n';
+import chrome from 'ui/chrome';
+import { BASE_PATH } from '../common/constants';
+
+if (chrome.getInjected('ccrUiEnabled')) {
+  const esSection = management.getSection('elasticsearch');
+
+  esSection.register('ccr', {
+    visible: true,
+    display: i18n.translate('xpack.ccr.appTitle', { defaultMessage: 'Cross Cluster Replication' }),
```

Comment:
__nit:__ aaaand one more prefix `xpack.ccr` :) 

Btw, we usually recommend to call `translate` within a function so that it's not called at module import time when there is a chance that plugin translations aren't initialized yet. I think in this case it will work, but if you can wrap it into a function that is called only when it's needed it'd be better.